pkg/controller/infinispan/util/configuration.go

```diff
@@ -58,7 +60,9 @@ func CreateInfinispanConfiguration(name string, xsite *XSite, loggingCategories
 		ClusterName: name,
 		JGroups:     jgroups,
 	}
-
+	if os.Getenv("JGROUPS_DIAGNOSTICS") == "TRUE" {
```

Comment:
@rigazilla looks like we should uppercase output value, as **TRUE** equals **true**

Reply:
Another one, should this variable be at the operator level or cluster site? A little bit confusing.

Reply:
it would be better if we can set it in the Infinispan CRD.

Reply:
Yes, true, @pruivo but we need to add a separate field for options like this (not just for single options but for future use also) or somehow to use InfinispanLoggingSpec.Categories.

Reply:
We added this configuration envvar for developing and it's not something required atm, but we can design a feature issue for the next release.
Does diagnostic expose any sensitive data? Security issues?

Reply:
@rigazilla wdyt about this?

Reply:
@rigazilla support team can also use this for diagnostic. Let's keep it "as is" but use uppercase for easy setup.

Reply:
@rigazilla please add uppercase func for os.Getenv

Reply:
uh, I though I did, let me check

Reply:
yeah @dmvolod it's there, GH is showing an outdated snippet